data_rows/QueryHistory.js

```diff
@@ -0,0 +1,147 @@
+import uuid4 from "uuid/v4";
+
+import DataRow from "./DataRow";
+import { LoginHistory } from "./LoginHistory";
+import { TYPES } from "./Types";
+import * as helpers from "../helpers";
+
+/**
+ * QUERY_HISTORY View
+ *
+ * https://docs.snowflake.net/manuals/sql-reference/account-usage/query_history.html
+ */
+export class QueryHistory extends DataRow {
+  constructor(databaseName, date) {
+    super();
+    this.QUERY_ID = uuid4();
+    this.setQueryText();
```

Comment:
in DatabaseStorageUsageHistory you use the `_convention` to annotate private methods. that's not super standard for node, but I like it.

Reply:
oops, sorry for lack of consistency. I'll switch everything applicable to "private"